src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -704,6 +704,32 @@ class CommonVisitor : public AST::BaseVisitor<Struct> {
             call_args_vec.p[arg_pos].loc = expr->base.loc;
             call_args_vec.p[arg_pos].m_value = expr;
         }
+        // Filling missing arguments with their defaults passed in function definition (if present).
+        std::string missed_args_names;
+        size_t missed_args_count =0;
+        for(size_t i = 0; i < orig_func->n_args; i++ ){
+            if(call_args_vec.p[i].m_value == nullptr){
+                ASR::Variable_t* var = ASRUtils::EXPR2VAR(orig_func->m_args[i]);
+                if (var->m_symbolic_value == nullptr){
+                    missed_args_names+="'" + (std::string) var->m_name + "' and ";
+                    missed_args_count++;
+                }
+                else{
```

Comment:
```suggestion
                    missed_args_names+="'" + std::string(var->m_name) + "' and ";
                    missed_args_count++;
                } else{
```

Reply:
@assem2002 it would be great if you could update the above to follow the existing style/formatting.